autoscaler/README.md

```diff
@@ -0,0 +1,37 @@
+# Horizontal Self Scaler container
+
+This container image watches over the number of schedulable nodes in the cluster and resizes
+the number of replicas in the required controller.
+
+Usage of pod_nanny:
```

Comment:
I like cluster-proportional-autoscaler (or autoscaler-by-cluster-size) because I think people are going to be confused about this vs. HPA and those names make the distinction clearer.